Objects/typeobject.c

```diff
@@ -11315,21 +11338,60 @@ update_slot(PyTypeObject *type, PyObject *name)
 
 /* Store the proper functions in the slot dispatches at class (type)
    definition time, based upon which operations the class overrides in its
-   dict. */
-static void
+   dict. Returns -1 and exception set on error or 0 otherwise.*/
+static int
 fixup_slot_dispatchers(PyTypeObject *type)
 {
+    int res = 0;
+
     // This lock isn't strictly necessary because the type has not been
     // exposed to anyone else yet, but update_ont_slot calls find_name_in_mro
     // where we'd like to assert that the type is locked.
     BEGIN_TYPE_LOCK();
 
+    PyObject *mro = Py_NewRef(lookup_tp_mro(type));
+
+    // Try to prebuild MRO dict. We build it in bottom-top manner,
```

Comment:
"Try to prebuild MRO dict." it's not an attempt, if an error occurs, everything is aborted.

```suggestion
    // Build MRO dict. We build it in bottom-top manner,
```

Reply:
Thanks!